src/query/boost_query.rs

```diff
@@ -20,6 +20,16 @@ impl BoostQuery {
     pub fn new(query: Box<dyn Query>, boost: Score) -> BoostQuery {
         BoostQuery { query, boost }
     }
+
+    /// The wrapped query.
+    pub fn query(&self) -> Box<dyn Query> {
```

Comment:
```rust
    pub fn underlying_query(&self) -> &dyn Query {
      ...
    }
```

Reply:
explanation: `query` is not descriptive enough, cloning by default is not necessary, Query inherits QueryClone